internal/olm/operator/bundle/install.go

```diff
@@ -87,6 +117,46 @@ func (i *Install) setup(ctx context.Context) error {
 		return err
 	}
 
+	// get index image labels.
+	catalogLabels, err := registryutil.GetImageLabels(ctx, nil, i.IndexImageCatalogCreator.IndexImage, false)
+	if err != nil {
+		return fmt.Errorf("get index image labels: %v", err)
+	}
+
+	// set the field to true if FBC label is on the image or for a default index image.
+	if _, hasFBCLabel := catalogLabels[containertools.ConfigsLocationLabel]; hasFBCLabel || i.IndexImageCatalogCreator.IndexImage == registry.DefaultIndexImage {
+		i.IndexImageCatalogCreator.HasFBCLabel = true
```

Comment:
When the index image is the default one, we'd still be making this API call to containertools even though it would have no effect on the if block and codepath. Ideally, we could save _some_ time by avoiding this call.

Maybe we can do this in a future improvement. What do you think?

Reply:
I guess the intention for this is that for the default index image, we want to default to FBC creation and not the old sqlite route. I'm assuming you're referring to the API calls to Operator Registry.

Reply:
`containertools.ConfigsLocationLabel` on line 127 is an API call, correct? However, if the index image is the default one, the `_, hasFBCLabel := catalogLabels[containertools.ConfigsLocationLabel]; hasFBCLabel` part of the if does not matter since we know we want to use FBC for the default index image. However, even though it does not matter, the API call is being made in the current implementation. I was thinking we wouldn't need to check what the label on the image is if the image is the default index image. This might be as simple as reversing the order of the two conditions that are joined by `||`.

Reply:
I see your point. Although I agree with you, I don't think it's syntactically possible to switch these two conditions as the second condition `_, hasFBCLabel := catalogLabels[containertools.ConfigsLocationLabel]; hasFBCLabel` is a map lookup which is combination of a condition (evaluates `hasFBCLabel` which will be true if key exists in the map) and initialization (which is setting `hasFBCLabel` to true). 

In my opinion, the only way to do this is invoke the API call and set `hasFBCLabel` before this if statement, and we can do something like `if i.IndexImageCatalogCreator.IndexImage == registry.DefaultIndexImage || hasFBCLabel`, but it still wouldn't avoid the API call.

Reply:
Alright, we can leave it as is.